bridge-ui/src/components/bridge/claiming/bridge-mode/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import styles from "./bridge-mode.module.scss";
+import Image from "next/image";
+import { useFormStore } from "@/stores";
+import { BridgeProvider } from "@/types";
+
+export default function BridgeMode() {
+  const token = useFormStore((state) => state.token);
+  const label = token.bridgeProvider === BridgeProvider.NATIVE ? "Native bridge" : "CCTP";
```

Comment:
if we had other bridge providers this conditional could get messy - a later refactor may have a provider to image mapping somewhere

Reply:
yea if we have more providers we can refactor into a small function, i.e. `const label = getBridgeModeLabel(token.bridgeProvider)`